ilock/__init__.py

```diff
@@ -56,7 +58,10 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
         if sys.platform.startswith('linux'):
             # In Linux you can delete a locked file
-            os.unlink(self._filepath)
+            try:
+                os.unlink(self._filepath)
+            except FileNotFoundError:

```

Comment:
Good change that I want to add too.
In rare cases that happen from time to time, its happen.

Reply:
I explained why this wouldn't work in https://github.com/symonsoft/ilock/pull/7#issuecomment-785545466.